### PR TITLE
issue #72: implemented posting system, added styling and created the …

### PR DESCRIPTION
**Pull request by TabrisArg**
issue #72 

* [X] Implemented Jekyll's posting system in the workshops page
* [X] Modified the main CSS files so posts will be affected by the stylesheet
* [X] Created extra file structure (_posts and assets/workshops folders) in order to support the posting system
* [X] Wrote the necessary documentation for the posting process

### CODE REVIEW

File: _layouts/post.html [hunk 1]
```diff
@@ -0,0 +1,10 @@
+{% include header.html %}
```
Comment (andiechu) [inline _layouts/post.html]: why not just use base.html?

File: _layouts/workshops.html [hunk 2]
```diff
@@ -0,0 +1,4 @@
+{% include header.html %}
```
Comment (andiechu) [inline _layouts/workshops.html]: same question, base.html?

File: _includes/header.html [hunk 3]
```diff
@@ -1,7 +1,7 @@
 <html lang="en">
 <head>
     <meta charset="utf-8">
-    <title>Workshops</title>  
+    <title> page.title</title>
```
Comment (andiechu) [inline _includes/header.html]: should be {{ page.title }}

File: _layouts/post.html (hunk 1 above)
Comment (TabrisArg) [reply _layouts/post.html]: I replaced base.html and eddited the respective layout

File: _layouts/workshops.html (hunk 2 above)
Comment (TabrisArg) [reply _layouts/workshops.html]: I replaced base.html and eddited the respective layout

File: _includes/header.html (hunk 3 above)
Comment (TabrisArg) [reply _includes/header.html]: Done

**Pull request merged**